### ISSUE TITLE
Question about line diff on PR

### ISSUE THREAD

**Issue by cdh981009** [snu-sf-class/swpp202201#64]
I have a question about how github (or git) PR line diff works

assume I implement a naive version of A in one PR, and then upgrade A to A' in the same sprint, and create two separate PRs,
does github recalculate line diff of PR A' after PR A is merged? Then line diff restriction in our project is applied on the changed line diff?

Thank you.

**Comment by strikef** [comment]
You're right. GitHub recalculates the line diff whenever there's a change in the merge target branch (which would be the main branch in our case). Therefore, the line diff restriction applies to the diff right before the merge.
